templates/account/signup.html

```diff
@@ -10,19 +10,8 @@ <h1>{% trans "Sign Up" %}</h1>
 
 <p>{% blocktrans %}Already have an account? Then please <a href="{{ login_url }}">sign in</a>.{% endblocktrans %}</p>
 
-<form class="signup" id="signup_form" method="post" action="{% url 'account_signup' %}">
-  {% csrf_token %}
-  {{ form.as_p }}
-  {% if redirect_field_value %}
-  <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
-  {% endif %}
-  <button type="submit">{% trans "Sign Up" %} &raquo;</button>
-</form>
-
-<div class="login-or">{% trans 'or' %}</div>
-
 {% get_providers as socialaccount_providers %}
-<p>Sign in with a third party account:</p>
+<p>Sign up with a third party account:</p>
```

Comment:
Can we comment these spots out (`<!--`) in case we need to bring them back? Or is it better to just pull from git history?

Reply:
Would rather just pull from the history